testing/README.md

```diff
@@ -0,0 +1,19 @@
+# Testing - Jupyter Accessibility
```

Comment:
Needs review

Reply:
I changed the name of the directory where all of this lives from "test" to "testing" because I think if the folder is named "test", many developers expect to find only test scripts in that directory. But I'm thinking this directory might hold more things than just test scripts. We might also want to put Isabela's human-readable tests in this directory, for example.

I felt that "testing" would still be easy to find for someone looking for tests.